tests/annotatedCommandTest.php

```diff
@@ -9,70 +9,72 @@
-    $this->setupModulesForTests($root);
-
-    // Enable our module. This will also clear the commandfile cache.
-    $this->drush('pm-enable', ['woot']);
-
-    // In theory this is not necessary, but this test keeps failing.
-    // $this->drush('cc', array('drush'), $options);
-
-    // Make sure that modules can supply DCG Generators and they work.
-    $optionsExample['answers'] = json_encode([
-      'name' => 'foo',
-      'machine_name' => 'bar',
-    ]);
-    $optionsExample['directory'] = self::getSandbox();
-    $original = getenv('SHELL_INTERACTIVE');
-    putenv('SHELL_INTERACTIVE=1');
-    $this->drush('generate', ['woot-example'], $optionsExample);
-    putenv('SHELL_INTERACTIVE=' . $original);
-    $target = Path::join(self::getSandbox(), '/src/Commands/ExampleBarCommands.php');
-    $actual = trim(file_get_contents($target));
-    $this->assertEquals('ExampleBarCommands says Woot mightily.', $actual);
-
-    // drush woot
-    $this->drush('woot');
-    $output = $this->getOutput();
-    $this->assertEquals('Woot!', $output);
-
-    // drush my-cat bet alpha --flip
-    $this->drush('my-cat', ['bet', 'alpha'], ['flip' => NULL]);
-    $output = $this->getOutput();
-    $this->assertEquals('alphabet', $output);
-
-    // drush my-cat bet alpha --flip
-    $this->drush('my-cat', ['bet', 'alpha'], ['flip' => NULL, 'ignored-modules' => 'woot'], NULL, NULL, self::EXIT_ERROR);
-
-    $this->drush('try-formatters');
-    $output = $this->getOutput();
-    $expected = <<<EOT
+    public function testGlobal()
+    {
+        $globalExtensions = $this->setupGlobalExtensionsForTests();
+
+        $options = [];
+
+        // We modified the set of available Drush commands; we need to clear the Drush command cache
+        $this->drush('cc', ['drush'], $options);
+
+        // drush foobar
+        $options['include'] = "$globalExtensions";
+        $this->drush('foobar', [], $options);
+        $output = $this->getOutput();
+        $this->assertEquals('baz', $output);
+
+        // Drush foobaz
+        $this->drush('foobaz', [], $options);
+        $output = $this->getOutput();
+        $this->assertEquals('bar', $output);
+    }
+
+    public function testExecute()
+    {
+        $sites = $this->setUpDrupal(1, true);
+        $uri = key($sites);
+        $root = $this->webroot();
+
+        // Copy the 'woot' module over to the Drupal site we just set up.
+        $this->setupModulesForTests($root);
+
+        // Enable our module. This will also clear the commandfile cache.
+        $this->drush('pm-enable', ['woot']);
+
+        // In theory this is not necessary, but this test keeps failing.
+        // $this->drush('cc', array('drush'), $options);
+
+        // Make sure that modules can supply DCG Generators and they work.
+        $optionsExample['answers'] = json_encode([
+        'name' => 'foo',
+        'machine_name' => 'bar',
+        ]);
+        $optionsExample['directory'] = self::getSandbox();
```

Comment:
Anyone know if we can convince phpcs to preserve indentation on these array elements?